Configs/.config/hypr/hyprland.conf

```diff
@@ -38,7 +38,7 @@ exec-once = wl-paste --type text --watch cliphist store # clipboard store text d
 exec-once = wl-paste --type image --watch cliphist store # clipboard store image data
 exec-once = $scrPath/swwwallpaper.sh # start wallpaper daemon
 exec-once = $scrPath/batterynotify.sh # battery notification
-
+exec-once = $scrPath/autostart.sh # run the startups
```

Comment:
Hi @ALEX5402,

This is good, but as I read through the discussion, both parties have good thoughts about this.

You can move this line to ./userprefs.t2 (https://github.com/prasanthrangan/hyprdots/blob/4c16d551524a41347608f0b609a60a49b372bb27/Configs/.config/hypr/userprefs.t2#L25) so we can make this optional.

I encourage you to make this optional as honestly this will become a burden for maintaining the repo, as by default we don't know what packages/services are run on startup.

Reply:
by default it gonna run those packages only which you enabled on startup from inside the app

Reply:
>  run those packages only which you enabled on startup from inside the app

Can you explain more about the script? What I understand of the script is it will extract the "Exec=" part, which is the command to launch  an app, nothing fancy.

What you are saying is the script can pinpoint which users enabled the startup "from inside the app" ? 
Or you are saying that ` $AUTOSTART_DIR ` is respected by most apps, and enabling it to run at the startup "from inside the app". Which I assumed that the application adds the .desktop file entry to the $AUTOSTART_DIR?

Reply:
ok

then should i change that with `$AUTOSTART_DI` ?